openapi/components/schemas/Applications/ApplicationInstanceConfiguration.yaml

```
@@ -0,0 +1,37 @@
+type: object
+required:
+  - settings
+description: |-
+  Application instance configuration.
+
+  Only user can change settings when `configuredBy` is `user`.
+
+  Only the application itself can change settings when `configuredBy` is `application`.
+properties:
+  settings:
+    description: Settings that are available for the user to complete when they install the application.
+    type: object
+    additionalProperties:
+      type: string
+    example:
+      color: red
+      level: gold
+  _links:
+    readOnly: true
+    type: array
+    items:
+      type: object
+      properties:
+        rel:
+          type: string
+          enum:
+            - self
+            - configuration
+          x-enumDescriptions:
+            self: Link to the application instance configuration
+            configuration: |-
+              Link to the external resource where application instance must be configured.
+              Contains JWT in query parameter.
+              Navigate user to this URL where application settings must be submitted.
+        href:
+          type: string
```

nitpick: missing description
```suggestion
          type: string
          description: Link URL.
```